### ISSUE TITLE
Mismatched tags line 132 column 2

### ISSUE THREAD

**Issue by ReyterYT** [LordOfPolls/Rule34-API-Wrapper#21]
i started using this module again after some configuration in my network to finally access rule34 smh, the problem is when i do something simple as 
```py
bot.rule34.getImages("tag") #bot.rule34 is a rule34.Rule34 instance
```

somehow i get this error
```py
  File "<string>", line None
xml.etree.ElementTree.ParseError: mismatched tag: line 132, column 2 
```

and im not quite sure how this happens

my rule34 version is  *1.7.5*
and my python version is *3.9*

_note: im using that with Jishaku so the error seems weird i know_

**Comment by LordOfPolls** [comment]
What do you mean, you had to do configuration on your network to get access to rule34? Are you sure you're accessing rule34.xxx?

That error looks like the xml the wrapper is getting isn't the one its expecting, and based on my testing here, it seems localised to you

**Comment by ReyterYT** [comment]
so im guessing this only happened to me?

**Comment by ReyterYT** [comment]
okay it seems that its cleared out that there was something redirecting it so this might be another embarrassing issues i made, have a good day

TL;DR
_my network automatically redirect it somewhere else_

**Comment by ReyterYT** [comment]
also why are you enforcing the `loop` parameter, some async module/packages already deprecated that parameter so why this one does not?

**Comment by LordOfPolls** [comment]
Glad you managed to resolve the issue. 

Apologies if you already know this, but for sake of completion ill explain anyways. 
Basically in versions of python before 3.6, ``get_event_loop`` was very unreliable for getting the currently active event loop. After 3.6 (irrc) this was rectified. This library, however, was written pre-3.6 and Ive never gotten around to updating it. Thankfully requiring the developer to pass ``loop`` doesn't cause any issues, besides a few extra characters of code

I'll probably update the library to make it optional at some point.

**Comment by ReyterYT** [comment]
ah alright then, yeah it doesn't bother im just curious why, well im looking forward to that update and have a great day 👋
